packages/eslint-config-airbnb/rules/react.js

```diff
@@ -8,12 +8,22 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     'react/display-name': 0,
+    // Forbid certain propTypes (any, array, object)
+    'react/forbid-prop-types': 0,
     // Enforce boolean attributes notation in JSX
     'react/jsx-boolean-value': 2,
+    // Validate closing bracket location in JSX
+    'react/jsx-closing-bracket-location': 2,
```

Comment:
tag-aligned would be the default. I guess we should add the config if we're adding a config to all rules.